requests/utils.py

```diff
@@ -38,6 +39,92 @@
 DEFAULT_CA_BUNDLE_PATH = certs.where()
 
 
+class _ListNode(object):
```

Comment:
This also doesn't belong in requests/utils.py as far as I'm concerned. We have a separate file for internal things like this, I think structs.py